File: java/src/main/java/model/params/TickerParams.java
package model.params;

import model.ChannelTypes;

public class TickerParams extends Params {

    private Float frequency;

    public TickerParams(String symbol, Float frequency) {
        super(ChannelTypes.TICKER, symbol);
        this.frequency = frequency;
    }

    public Float getFrequency() {
        return frequency;
    }

    public void setFrequency(Float frequency) {
        this.frequency = frequency;
    }

    @Override
    public String toString() {
        return "TickerParams{" +
                super.toString() +
                ", frequency=" + frequency +
                "} ";
    }
}

File: c-sharp/ws-client/model/parameters/TickerParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_client.model.parameters
{
    class TickerParams : Params
    {
        public TickerParams(string symbol, float frequency) : base(ChannelTypes.TICKER, symbol)
        {
            this.frequency = frequency;
        }

        public float frequency { set; get; }
    }
}
